### PR TITLE
Update Texts for negative RAT

### PR DESCRIPTION
**Pull request by marcussc**
## Description
This PR changes the description on a negative RAT in english.

## Screenshots
Screen Shot 2021-04-30 at 11 28 15

### CODE REVIEW

**Review by freshking — APPROVED** [review]

**Review by naveeddotio — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -845,9 +845,9 @@
 
 "ExposureSubmissionResult_Timer_Title" = "Result available since";
 
-"ExposureSubmissionResult_Negative_Antigen_Proof_Title" = "Verification Feature";
+"ExposureSubmissionResult_Negative_Antigen_Proof_Title" = "";
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:848]: 🤷‍♂️

**Review by whiskey — APPROVED** [review]
Whatever it takes…!

**Comment by KaiTeuber** [comment]
🍎 👏

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -845,9 +845,9 @@
...
 
-"ExposureSubmissionResult_Negative_Antigen_Proof_Desc" = "You can also use the test result displayed here as proof of a negative rapid test result. \n\nPlease note that you are only required to prove your rapid test result if this is required by law. You can prove the result using the app or in a different way. For more information, please familiarize yourself with the criteria for the acceptance of proof of testing in your federal state.";
+"ExposureSubmissionResult_Negative_Antigen_Proof_Desc" = "Your test result is private and you are not required to share it unless required by law.\nIf you choose to share it, you may show this screen, or use one of the many others ways provided.\nFor more information, please familiarize yourself with the criteria for verification of testing in your federal state.";
```
Comment (ChristinaTsigganos) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:850]:
```suggestion
"ExposureSubmissionResult_Negative_Antigen_Proof_Desc" = "Your test result is private and you are not required to share it unless required by law.\nIf you choose to share it, you may show this screen, or use one of the many other ways provided.\nFor more information, please familiarize yourself with the criteria for verification of testing in your federal state.";
```

**Review by pascal-brause — APPROVED** [review]

**Review by ChristinaTsigganos — CHANGES_REQUESTED** [review]
Please remove typo. When typo is fixed, this can be merged.

**Review by KaiTeuber — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

**Review by SwissMountainFrog — APPROVED** [review]
